www/content/_index.md

```diff
@@ -46,7 +46,7 @@ $ tricorder -i /path/to/inventory do -- echo "{host.id} says {host.vars.msg}"
 ## Add dependency
 
 ```toml
-tricorder = "0.9"
```

Comment:
Please don't put the next version (0.10.1) here, this should be done in a dedicated PR afterwards (I will probably automate this with a Github Actions workflow later on).

Reply:
ok